src/openms/source/ANALYSIS/ID/FragmentIndex.cpp

```diff
@@ -294,12 +462,25 @@ namespace OpenMS
     defaults_.setValidStrings("precursor_mz_tolerance_unit", tolerance_units);
     defaults_.setValue("fragment_mz_tolerance_unit", "DA", "Unit of tolerance for fragment-m/z");
     defaults_.setValidStrings("fragment_mz_tolerance_unit", tolerance_units);
-    defaults_.setValue("modifications_fixed", std::vector<std::string>{}, "Fixed modifications, specified using UniMod (www.unimod.org) terms, e.g. 'Carbamidomethyl (C)'");
+    defaults_.setValue("modifications_fixed", std::vector<std::string>{"Carbamidomethyl (C)"}, "Fixed modifications, specified using UniMod (www.unimod.org) terms, e.g. 'Carbamidomethyl (C)'");
     defaults_.setValidStrings("modifications_fixed", ListUtils::create<std::string>(all_mods));
-    defaults_.setValue("modifications_variable", std::vector<std::string>{}, "Variable modifications, specified using UniMod (www.unimod.org) terms, e.g. 'Oxidation (M)'");
+    defaults_.setValue("modifications_variable", std::vector<std::string>{"Oxidation (M)"}, "Variable modifications, specified using UniMod (www.unimod.org) terms, e.g. 'Oxidation (M)'");
     defaults_.setValidStrings("modifications_variable", ListUtils::create<std::string>(all_mods));
     defaults_.setValue("max_variable_mods_per_peptide", 2, "Maximum number of residues carrying a variable modification per candidate peptide");
     is_build_ = false; // TODO: remove this and build on construction
+
+    //Search-related params
+
+    defaults_.setValue("Min_matched_peaks", 5, "Minimal number of matched ions to report a PSM");
+    defaults_.setValue("min_isotope_error", 0, "Precursor isotope error");
+    defaults_.setValue("max_isotope_error", 0, "precursor isotope error");
+    defaults_.setValue("min_precursor_charge", 1, "min precursor charge");
+    defaults_.setValue("max_precursor_charge", 1, "max precursor charge");
+    defaults_.setValue("max_fragment_charge", 1, "max fragment charge");
```

Comment:
if max_fragment_charge = 2 does the used charge depend on the actual precursor charge?
I think in general it is not clear if these parameters configure how the fragment index is built, what is related to the query and what is related to the peptide search.